src/services/standardGoals.ts

```diff
@@ -75,6 +75,7 @@ export async function removeObjectivesFromReport(objectivesToRemove, reportId) {
     where: {
       id: objectivesToDefinitelyDestroy.map((o) => o.id),
     },
+    individualHooks: true, // We need this to ensure onAR is updated.
```

Comment:
These would fail to call recalcuate on AR when we removed the goal or objective from the AR.